docs/ui/components/BoxLink/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { css } from '@emotion/react';
 import { borderRadius, spacing, theme, ArrowRightIcon, iconSize, shadows } from '@expo/styleguide';
-import React, { PropsWithChildren, ReactNode } from 'react';
+import { IconProps } from '@expo/styleguide/dist/types';
```

Comment:
```suggestion
import type { IconProps } from '@expo/styleguide/dist/types';
```
Not sure if we use that in docs, but we do everywhere else afaik

Reply:
Yeah, we can use type-only import here, it's a good practice, nice catch!

Also, we need to export the `IconProps` correctly at some point. 😉